src/data.py

```diff
@@ -247,20 +252,40 @@ def parse_static_op_hours(hours_list, eatery_id, dining_items):
   new_operating_hours = []
   for i in range(NUM_DAYS_STORED_IN_DB):
     new_date = today + timedelta(days=i)
-    new_events = weekdays.get(new_date.weekday(), [])
 
-    for event in new_events:
-      event['menu'] = dining_items
+    for dates in dates_closed:
+      if '-' not in dates:
+        closed_date = datetime.strptime(dates, '%m/%d/%y').date()
+        if is_date_in_range(new_date, closed_date, closed_date):
```

Comment:
is this just `new_date == closed_date`?

Reply:
mmm yeaaah

Reply:
should I just drop the helper all together and replace both uses of the function with the appropriate boolean?

Reply:
yeah that would be good lol

Reply:
lol so basically just moved the whole initial helper I made into another function